trustymail/domain.py

```diff
@@ -41,12 +41,34 @@ def __init__(self, domain_name):
         # Mail Info
         self.mail_servers = []
 
+        # A string for each port for each entry in mail_servers indicating
+        # whether or not the server sends mail
+        self.sends_mail = []
+
+        # A string for each port for each entry in mail_servers indicating
+        # whether or not the server supports STARTTLS
+        self.starttls = []
+
         # A list of any errors that occurred while scanning records.
         self.errors = []
 
     def has_mail(self):
         return len(self.mail_servers) > 0
 
+    def has_sends_mail(self):
+        """
+        Returns True if information about whether mail servers send
+        mail is present and otherwise returns False.
+        """
+        return len(self.starttls) > 0
+
+    def has_starttls(self):
+        """
+        Returns True if STARTTLS information is present and otherwise
```

Comment:
I think `Has STARTTLS` should be a Boolean indicating True when (at least one port? all ports?) use STARTTLS. The raw findings should probably be called something different the `Supports STARTTLS`; the nomenclature other fields in trustymail follow for this type of output is "_x_ Results", like `STARTTLS results` .

Reply:
That was exactly my confusion.  Should "Has STARTTLS" be true if the IP listens on 25 and 587 but only supports STARTTLS on 25?  

I'm happy to split "Has STARTTLS" into a Boolean field and a results field, but I need a little guidance on what to do when one port supports STARTTLS but another (active) port does not.